src/ActionList/ActionList.stories.tsx

```diff
@@ -0,0 +1,279 @@
+export default {
+  title: 'Components/ActionList',
+  component: ActionList,
+  subcomponents: {Item, LinkItem, Group, Divider, Description}
+} as Meta<typeof ActionList>
+
+export const Default = () => (
+  <ActionList>
+    <ActionList.Item>Copy link</ActionList.Item>
+    <ActionList.Item>Quote reply</ActionList.Item>
+    <ActionList.Item>Edit comment</ActionList.Item>
+  </ActionList>
+)
+
+export const Playground: Story<ActionListProps> = args => (
+  <ActionList {...args}>
+    <ActionList.Item>Copy link</ActionList.Item>
+    <ActionList.Item>Quote reply</ActionList.Item>
+    <ActionList.Item>Edit comment</ActionList.Item>
+  </ActionList>
+)
+Playground.args = {
+  showDividers: false,
+  selectionVariant: undefined,
+  variant: 'inset',
+  role: 'listbox'
+}
+Playground.argTypes = {
+  showDividers: {
+    control: {
+      type: 'boolean'
+    }
+  },
+  variant: {
+    control: {
+      type: 'radio'
+    },
+    options: ['inset', 'full']
+  },
+  selectionVariant: {
+    control: {
+      type: 'radio',
+      labels: ['single', 'multiple', 'unset']
+    },
+    options: [0, 1, 2],
+    mapping: ['single', 'multiple', null]
+  },
+  role: {
+    type: 'string'
+  }
+}
+
+const icons = ['unset', 'TypographyIcon', 'VersionsIcon', 'SearchIcon', 'ArrowRightIcon', 'ArrowLeftIcon']
+
+// @ts-ignore ignoring types here to pass in options for icon selection in Storybook
+export const ItemPlayground = args => {
+  let leadingVisual
+  if (args.leadingVisual === 'TypographyIcon') {
+    leadingVisual = <TypographyIcon />
+  } else if (args.leadingVisual === 'VersionsIcon') {
+    leadingVisual = <VersionsIcon />
+  } else if (args.leadingVisual === 'SearchIcon') {
+    leadingVisual = <SearchIcon />
+  } else if (args.leadingVisual === 'ArrowRightIcon') {
+    leadingVisual = <ArrowRightIcon />
+  } else if (args.leadingVisual === 'ArrowLeftIcon') {
+    leadingVisual = <ArrowLeftIcon />
+  } else if (args.leadingVisual === 'unset') {
+    leadingVisual = null
+  }
+
+  let trailingVisual
+  if (args.trailingVisual === 'TypographyIcon') {
+    trailingVisual = <TypographyIcon />
+  } else if (args.trailingVisual === 'VersionsIcon') {
+    trailingVisual = <VersionsIcon />
+  } else if (args.trailingVisual === 'SearchIcon') {
+    trailingVisual = <SearchIcon />
+  } else if (args.trailingVisual === 'ArrowRightIcon') {
+    trailingVisual = <ArrowRightIcon />
+  } else if (args.trailingVisual === 'ArrowLeftIcon') {
+    trailingVisual = <ArrowLeftIcon />
+  } else if (args.trailingVisual === 'unset') {
+    trailingVisual = null
+  }
+
+  return (
+    <ActionList selectionVariant={args.selectionVariant}>
+      <ActionList.Item {...args}>
+        {leadingVisual && <ActionList.LeadingVisual>{leadingVisual}</ActionList.LeadingVisual>}
+        Action list item
+        {trailingVisual && <ActionList.TrailingVisual>{trailingVisual}</ActionList.TrailingVisual>}
+      </ActionList.Item>
+    </ActionList>
+  )
+}
+ItemPlayground.argTypes = {
```

Comment:
Missing `selectionVariant` here. I'm getting an error when using the `ItemPlayground` story + setting the `selected` control to `true`.

`For Item to be selected, ActionList or ActionList.Group needs to have a selectionVariant defined`



<img width="1820" alt="Screenshot 2022-11-07 at 16 05 25" src="https://user-images.githubusercontent.com/912236/200343540-f7e50bc6-fbdb-4a91-828e-3068d0ec2e41.png">

Reply:
Fixed! Thank you ✨